### PR TITLE
match keepalive response on transaction id

### LINKED ISSUE #10303: Shared computer /keepalive fix
VA.gov uses the "/keepalive" endpoint the IAM team created to determine whether or not the user has an active SSOe session.  In some cases it's possible, say using a shared computer, for a VA.gov user to retain access of another user who had previously signed in.  To mitigate this issue, we need the IAM team to update the "/keepalive" endpoint with a unique transaction value, so we can verify on the VA.gov side that the active SSOe session matches the VA.gov session.

See discussion here: https://dsva.slack.com/archives/CHPA76VV5/p1590534133354700

### PR DESCRIPTION
**Pull request by ericbuckley**
We need to use the transaction id value returned from the `/keepalive` endpoint and match that to the transaction id in the users VA.gov profile.  If they differ we need to immediately log the user out, as it could be a different user that is logged in.

deps https://github.com/department-of-veterans-affairs/vets-api/pull/4580
fixes https://github.com/department-of-veterans-affairs/va.gov-team/issues/10303

test cases to come....

### CODE REVIEW

**Review by patrickvinograd — APPROVED** [review]
LGTM

File: src/platform/utilities/sso/index.js [hunk 1]
```diff
@@ -33,23 +33,28 @@ export async function ssoKeepAliveSession() {
...
       // the user authenticated they used SSOe, otherwise we can't auto logout
-      // explicitly check to see if the TTL for the SSO3 session is 0, as it
+      // explicitly check to see if the TTL for the SSOe session is 0, as it
       // could also be null if we failed to get a response from the SSOe server,
       // in which case we don't want to logout the user because we don't know
+      // Additionally check to make sure that the transaction id returned from
+      // the /keepalive endpoint doesn't match that of the users VA session.
+      // If they don't match, it means we might have a different user logged in,
+      // thus we should auto log out the user, and immediately log them back in
+      // to make sure the users match
```
Comment (hinzed1127) [inline src/platform/utilities/sso/index.js:57]: This reads a little confusingly to me. I would reword to something like this:

> Additionally, compare the transaction id from the `keepalive` endpoint with the existing transaction id. If they don't match, it means we might have a different user logged in. Thus, we should auto log out the user, and immediately log them back in to make sure the users match.

or something like that. all of the negatives in the beginning read less clearly to me

**Review by hinzed1127 — APPROVED** [review]
LGTM pending the complementary API changes

**Review by bkjohnson — APPROVED** [review]
